utils/twarc-archive.py

```diff
@@ -44,9 +44,9 @@
 def main():
     e = os.environ.get
     parser = argparse.ArgumentParser("archive")
-    parser.add_argument("search", action="store",
+    parser.add_argument("search", type=commandline_unicode, action="store",
```

Comment:
I see this when running after applying these changes:

NameError: global name 'commandline_unicode' is not defined